plans.go

```diff
@@ -29,6 +30,7 @@ type ListPlansParams struct {
 	ExternalID     string `json:"external_id,omitempty"`
 	System         string `json:"system,omitempty"`
 	Cursor
+	PaginationWithCursor
```

Comment:
The `per_page` is tagged once in `Cursor`, once in `PaginationWithCursor`. What's the consequence?